forge-gui/res/cardsfolder/l/lazav_dimir_mastermind.txt

```diff
@@ -4,5 +4,5 @@ Types:Legendary Creature Shapeshifter
 PT:3/3
 K:Hexproof
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Graveyard | ValidCard$ Creature.nonToken+OppOwn | TriggerZones$ Battlefield | Execute$ LazavCopy | OptionalDecider$ You | TriggerDescription$ Whenever a creature card is put into an opponent's graveyard from anywhere, you may have CARDNAME become a copy of that card, except its name is Lazav, Dimir Mastermind, it's legendary in addition to its other types, and it has hexproof and this ability.
-SVar:LazavCopy:DB$ Clone | Defined$ TriggeredCard | NewName$ Lazav, Dimir Mastermind | AddTypes$ Legendary | AddKeywords$ Hexproof | GainThisAbility$ True | Optional$ True | AddSVars$ LazavCopy | AILogic$ IfDefinedCreatureIsBetter
+SVar:LazavCopy:DB$ Clone | Defined$ TriggeredCardLKICopy | NewName$ Lazav, Dimir Mastermind | AddTypes$ Legendary | AddKeywords$ Hexproof | GainThisAbility$ True | Optional$ True | AddSVars$ LazavCopy | AILogic$ IfDefinedCreatureIsBetter
```

Comment:
@Hanmac
we might need a follow-up for cases like this:
- e. g. _Clone_ dies and in response to this trigger Reanimate is used on it
- now Lazav would copy the battlefield Clone (because we don't create a new object)
- but it should have used LKI from Graveyard instead (and then most likely die from 0/0)

Reply:
So should NewCard also be an LKI Copy of the Graveyard Card? 🤔

Reply:
well the problem is this **not** being an always "look back in time" thing
so creating LKI when it triggers wouldn't be right, as long as the card stays in GY it should see the current state
does it make a difference for the copiable values? maybe not in this case but others might care

Reply:
`leftGraveyardThisTurn` would have LKI from the **right moment**
but it would need some logic to get used there